Volume/Models/WeeklyDebrief.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  WeeklyDebrief.swift
+//  Volume
+//
+//  Created by Hanzheng Li on 11/17/21.
+//  Copyright © 2021 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+
+struct WeeklyDebrief: Codable {
+    let creationDate: Date
+    let expirationDate: Date
+    let numShoutouts: Int
+    let numReadArticles: Int
+    let numBookmarkedArticles: Int
+    let readArticleIDs: [ArticleID]
+    let randomArticleIDs: [ArticleID]
+    
+    init(from weeklyDebrief: WeeklyDebriefFields) {
+        /*
+         schema changes required:
+         - createdAt -> creationDate
+         - numShoutouts, numReadArticles, numBookmarkedArticles -> type Int
+         */
```

Comment:
Why weren't these schema changes made already? We shouldn't be committing work-in-progress code to main.